bears/general/LineLengthBear.py

```diff
@@ -5,15 +5,33 @@
 from coalib.bears.LocalBear import LocalBear
 from coalib.results.Result import Result
 from coalib.settings.Setting import typed_list
+from coalib.bearlib.aspects import map_setting_to_aspect
+from coalib.bearlib.aspects.Formatting import (
+    Formatting,
+    LineLength,
+    Indentation,
+)
 
 
-class LineLengthBear(LocalBear):
+class LineLengthBear(
+        LocalBear,
+        aspects={
+            'detect': [
+                LineLength,
+                Indentation,
+            ]},
+        languages=['python'],
```

Comment:
Below it says `LANGUAGES = {'All'}`.

And the logic of this bear should work with any language.

I dont believe we have any language properties which help with these settings.

Can we drop languages here?

I will need to see some new tests for this bear which use aspects in the settings; that will help us understand how this works.

Reply:
We can't drop it. I don't know why but bear classes accept either 1 or 3 arguments. If I don't specify the language, it gives an error. We need a list of all languages here. I see most of the language files are imported here https://github.com/coala/coala/blob/master/coalib/bearlib/languages/__init__.py, can we import rest of the files and make a function to return a list of all languages (in a separate PR)?

Reply:
Yes, we need an object which represents 'all languages' (including `Unknown`) which can be imported and used here.